packages/react-vapor/src/components/actions/TriggerActionConnected.tsx

```diff
@@ -81,4 +81,7 @@ export class TriggerAction extends React.Component<ITriggerActionProps, any> {
     }
 }
 
-export const TriggerActionConnected = connect(undefined, mapDispatchToProps)(TriggerAction);
+export const TriggerActionConnected = connect<null, ReturnType<typeof mapDispatchToProps>, ITriggerActionOwnProps>(
+    undefined,
+    mapDispatchToProps
+)(TriggerAction as any);

```

Comment:
I cannot use because sometime the non-connected version of the component is used (e.g., `<TriggerAction action={...} />`) It's the only I found of making it work